src/Hosting/Hosting/src/Server/Features/ServerAddressesFeature.cs

```diff
@@ -7,10 +7,19 @@
 
 namespace Microsoft.AspNetCore.Hosting.Server.Features
 {
+    /// <summary>
+    /// Specifies the address used by the server.
+    /// </summary>
     public class ServerAddressesFeature : IServerAddressesFeature
     {
+        /// <summary>
+        /// A <see cref="ICollection{T}" /> of addresses used by the server.
+        /// </summary>
         public ICollection<string> Addresses { get; } = new List<string>();
```

Comment:
I think you should be able to use `<inheritdoc />` here. It will use the description from the interface.

Reply:
Done.